### PR TITLE
Update uuidv4 import

### PR DESCRIPTION
**Pull request by anuranduttaroy**
**Issue #:** https://github.com/aws/amazon-chime-sdk-js/issues/437

**Description of changes:**
Updated version (_"uuid": "^8.2.0"_) of **uuidv4** and how is imported. More info: https://www.npmjs.com/package/uuid **Version: 8.2.0**

**Testing**

1. Have you successfully run `npm run build:release` locally?
Yes.

2. How did you test these changes?
Manually ran a few integration tests (which were updated) and it worked as expected. The tests succeeded.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,7 +66,8 @@
     "@types/dom-mediacapture-record": "^1.0.4",
     "detect-browser": "^4.7.0",
     "protobufjs": "~6.8.8",
-    "resize-observer": "^1.0.0"
+    "resize-observer": "^1.0.0",
+    "uuid": "^8.2.0"
```
Comment (simmkyu) [inline package.json:70]: Chime SDK doesn't need to depend on the `uuid` library. Update only in the integration and demo's package.json.

Comment (anuranduttaroy) [reply package.json]: Fixed.

**Comment by simmkyu** [comment]
@anuranduttaroy Before merging PR, could you confirm if all demos with new UUID work okay?
I recommend a fresh setup: cloining this repo, checking out your branch `uuid-v4-fix`, and ensuring that demos work.

**Comment by anuranduttaroy** [comment]
> @anuranduttaroy Before merging PR, could you confirm if all demos with new UUID work okay?
> I recommend a fresh setup: cloining this repo, checking out your branch `uuid-v4-fix`, and ensuring that demos work.

Yes, let me do that.

File: demos/serverless/src/handlers.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 const AWS = require('./aws-sdk');
 const fs = require('fs');
-const uuid = require('./uuid/v4');
+const { v4: uuidv4 } = require('uuid');
```
Comment (simmkyu) [inline demos/serverless/src/handlers.js:6]: The deploy script is installing modules and copying `aws-sdk` and `uuid` packages to the local folder. I think you should keep loading it from locally but changing the left-hand only. Please try running the script in a fresh repo.

https://github.com/aws/amazon-chime-sdk-js/blob/a37d6d82208c79e79a44f8d9ed4c2e26539d8d3a/demos/serverless/deploy.js#L144-L148

Comment (anuranduttaroy) [reply demos/serverless/src/handlers.js]: Updated: `const { v4: uuidv4 } = require('./uuid');`

Tested with both the videohelpdesk & serverless demo.

**Pull request merged**